articles/keywords-and-identifiers.html

```diff
@@ -114,14 +115,20 @@ <h3>Identifiers</h3>
 エラー
 </code></pre>
 
-And here are some tokens which are illegal to be used as identifers:
-<pre class="line-numbers"><code class="language-go">123
+And here are some tokens which are illegal to be used as identifiers:
+<pre class="line-numbers"><code class="language-go">// Starting with a unicode digit.
+123
 3apples
+
+// Containing illegal unicode characters.
 a.b
 *ptr
 $name
 a@b.c
-type // this is a keyword
+
+// It is a reserved keyword.
+type
+int

```

Comment:
I noticed you add an "int" here.
But "int" is really a good identifier. It is a predeclared identifier which can be shadowed by our custom ones.

Also the "reserved" wording is unnecessary. There are no reserved keywords in Go.

Reply:
You are right about `int`.

But I don't get the second part!
> There are no reserved keywords in Go.
What do you mean?